activation/post_verifier.go

```diff
@@ -365,17 +373,17 @@ func (w *postVerifierWorker) start() {
 		// First try to process a prioritized job.
 		select {
 		case job := <-w.prioritized:
-			job.result <- w.verifier.Verify(job.ctx, job.proof, job.metadata, job.opts...)
+			job.result <- w.verifier.Verify(job.ctx, job.proof, job.metadata, WithVerifierOptions(job.opts...))
```

Comment:
Not a NIT but I'd say it's fine not to address this in this PR:
Blocking reads/writes to channels are generally discouraged and we should avoid using them in the codebase, as they can cause all sorts of deadlocks and [goroutine leaks](https://go.dev/wiki/CodeReviewComments#goroutine-lifetimes).
In general I think we should be using the pattern of using:
```go
select {
  case c<-val:
  case <-ctx.Done():
  default:
    // this is not always needed, but ensures there's handling of the edge case where we couldn't write to the channel but also the context hasn't expired
```
The side effect is that it makes the code a bit more verbose, but avoids all sorts of other anomalies.